### ISSUE TITLE
SIVIC proposed updates from player controls sub-group

### ISSUE THREAD

**Issue by jaeccles** [InteractiveAdvertisingBureau/SIMID#4]
Conversations within the player controls sub-group aligned on 3 recommendations, to be folded into this spec once its Github migration is complete (our other recommendations will most likely be absorbed into a larger video ad player Design Guide in a parallel work stream):

1. The SIVIC creative should be rendered in a layer that only overlays the video media element, and not any surrounding video player chrome or out-of-video player control UI elements. This will prevent easily avoidable overlapping clashes between native and creative UI elements, and allow for greater predictability in where and how SIVIC creative elements may overlay elements of the SIVIC video itself.
2. The SIVIC creative should have a lower z-index than any of the player UI. This means that, in the unfortunate scenario of overlap between video controls and the SIVIC creative, the user will at least not be blocked from controlling their standard video experience as they would otherwise expect.
3. Touches or clicks on one SIVIC layer should not carry into another layer. For example, if the user touches a player UI pause button, which happens to unfortunately overlay a button in the SIVIC creative, then that interaction should be treated as a "pause" and not as an interaction with the SIVIC creative button.

**Comment by ryanthompson591** [comment]
We have decided that this should go in a best practices guide and not be part of the spec.
